README.md

```diff
@@ -1,11 +1,14 @@
 is-ipfs
 ====
 
-[![build status](https://secure.travis-ci.org/ipfs/is-ipfs.svg)](http://travis-ci.org/ipfs/is-ipfs)
-[![dignified.js](https://img.shields.io/badge/follows-dignified.js-blue.svg?style=flat-square)](https://github.com/dignifiedquire/dignified.js)
+[![](https://img.shields.io/github/release/ipfs/ipfs-companion.svg)](https://github.com/ipfs/ipfs-companion/releases/latest)
```

Comment:
IPFS companion?

Reply:
Ooops, fixed in https://github.com/ipfs/is-ipfs/pull/18/commits/ae0f73809dc55989b5465626c30bdd9ad147e4ef